src/XrdSecgsi/XrdSecProtocolgsi.hh

```diff
@@ -188,7 +188,7 @@ public:
    char  *proxy;  // [c] user proxy  [/tmp/x509up_u<uid>]
    char  *valid;  // [c] proxy validity  [12:00]
    int    deplen; // [c] depth of signature path for proxies [0] 
-   int    bits;   // [c] bits in PKI for proxies [512] 
+   int    bits;   // [c] bits in PKI for proxies [2048]
```

Comment:
Same here, refer to the CryptoAux definition so that when it becomes 4096 we know where to look.